src/track/serato/markers.cpp

```diff
@@ -148,6 +148,13 @@ SeratoMarkersEntryPointer SeratoMarkersEntry::parseID3(const QByteArray& data) {
 
     stream >> colorSerato32 >> type >> isLocked;
 
+    if (isLocked && type != static_cast<quint8>(TypeId::Loop)) {
+        kLogger.warning() << "Parsing SeratoMarkersEntry failed:"
+                          << "isLocked field is not false for non-loop type"
+                          << static_cast<quint8>(TypeId::Loop);
```

Comment:
Is this really a fatal error or should we correct this? Please add a comment. 

The same below.

Reply:
Depends on what you consider a fatal error. I tried to be careful here, and therefore fail parsing every time something unexpected is encountered. Hopefully, if some of these assumptions are false, someone will open a bug report because Serato data is not imported.

Reply:
Ok, so can you add something like this to the code (in your words)? 

// We fail here bacause it is an unexpected situation that should not be corrected silently without knowing what we do. Write a bug if this assumption is wrong.